kernel/Kconfig

```diff
@@ -331,7 +331,7 @@ config BOOT_BANNER
 config BOOT_BANNER_STRING
 	string "Boot banner string"
 	depends on BOOT_BANNER
-	default "Booting Zephyr OS build"
+	default "Booting nRF Connect SDK"
```

Comment:
does this need to be a `noup`, can't we place the default overwrite in sdf-nrf here: https://github.com/nrfconnect/sdk-nrf/blob/main/Kconfig.nrf ?